common/lib/dependabot/update_checkers/base.rb

```diff
@@ -10,12 +10,12 @@ class Base
       attr_reader :dependency, :dependency_files, :repo_contents_path,
                   :credentials, :ignored_versions, :raise_on_ignored,
                   :security_advisories, :requirements_update_strategy,
-                  :options
+                  :group, :options
```

Comment:
Nit: should we prefer `dependency_group` as a hint to the base class? 

I'm conscious that some package managers have their own concept of a group so we might eventually do more do have `dependency_group` look at `native_group` or `package_manager_group` in future.